docs/docs/remark-plugin-tutorial.md

```diff
@@ -0,0 +1,283 @@
+---
+title: "Remark Plugin Tutorial"
+---
+
+[`gatsby-transformer-remark`](/plugins/gatsby-transformer-remark) empowers developers to translate Markdown into HTML to be consumed via Gatsby's GraphQL API. Blogs and other content based sites can highly benefit from functionality enabled with this plugin. With this plugin, authors of content for the site don't need to worry about how the site is written or structured but can rather focus on writing engaging posts and content!
+
+In certain instances, a developer may want to customize the content of the Markdown file and extend it functionally in useful ways; for example, use cases such as [adding syntax highlighting](/packages/gatsby-remark-prismjs/), [parsing and creating responsive images](/packages/gatsby-remark-images), [embedding videos](/packages/gatsby-remark-embed-video), and many others. In each of these examples, a plugin will be injected with the Markdown Abstract Syntax Tree (AST) and manipulate content based on certain node types or content in particular nodes.
+
+## What you will learn in this tutorial
+
+- Further understanding of the remark Abstract Syntax Tree (AST)
```

Comment:
Note to self: add this to the glossary! That can come after https://github.com/gatsbyjs/gatsby/pull/13465.